vulture/core.py

```diff
@@ -91,14 +91,22 @@ def _ignore_variable(filename, varname):
         _is_special_name(varname))
 
 
-class Item(str):
-    def __new__(cls, name, typ, filename, lineno, size=1):
-        item = str.__new__(cls, name)
-        item.typ = typ
-        item.filename = filename
-        item.lineno = lineno
-        item.size = size
-        return item
+class Item(object):
+    def __init__(self, name, typ, filename, lineno, size=1):
+        self.name = name
+        self.typ = typ
+        self.filename = filename
+        self.lineno = lineno
+        self.size = size
+
+    def __str__(self):
+        return self.name
+
+    def __eq__(self, other):
+        return self.name == other
```

Comment:
That would require us to compare it with only `Item` objects.
 IMHO, this would cater to a better UX.
For ex, when making use of the API, it would be better if we could just compare it with the "name" string.